doc/craft/ratelimitpolicy.md

```diff
@@ -0,0 +1,255 @@
+# Crafting A RateLimitPolicy
+A detail explanation of the fields in the RateLimitPolicy is outlined here.
+For a quick reference refer to [Reference](/kuadrant-operator/doc/reference/ratelimitpolicy/) section.
+
+## apiVersion and kind
+The latest version of the RateLimitPolicy is `kuadrant.io/v1beta2`.
+With the kind being `RateLimitPolicy`.
+```yaml
+apiVersion: kuadrant.io/v1beta2
+kind: RatelimitPplicy
+```
+
+## Metadata
+When creating the RateLimitPolicy the policy must be crated in the same namespace as the resource being targeted.
+```yaml
+metadata:
+    name: toystore
+    namespace: default
+```
+Once created Kuadrant may add and manage additional annotations and labels.
+For example:
+```yaml
+metadata:
+    annotations:
+        kuadrant.io/namespace: default
+```
+
+## Spec
+
+### limits
+`spec.limits` is a list of named limits, which describe what limits should be placed on a resource.
+Many named limits are allowed.
+Limit names do require to be valid yaml keys.
+```yaml
+spec:
+    limits:
+        limit_1:
+            ...
+        limit_2:
+            ...
+```
+
+#### counters
+`counters` defines additional rate limit counters based on context qualifiers and well known selectors.
+The full list of well known attributes can be found [here](/architecture/rfcs/0002-well-known-attributes/#request-attributes).
```

Comment:
Broken link.

Also, pointing to one group of well-known attributes only (i.e. request attributes.) Attributes from the other groups also apply.